src/main/java/com/faforever/client/chat/ChatCategoryItemController.java

```diff
@@ -89,6 +90,14 @@ public void onCategoryClicked(MouseEvent mouseEvent) {
     }
   }
 
+  public void bindToUserList(ObservableList<ChatUserItem> userList) {
+    JavaFxUtil.bind(userCounterLabel.textProperty(), Bindings.size(userList).asString());
+  }
+
+  public void dispose() {
+    JavaFxUtil.unbind(userCounterLabel.textProperty());
+  }
```

Comment:
Technically bindings are weak references so don't know if this is necessary but not harmful